### PR TITLE
Adds a surgery to attach a chainsaw to your hand

### PR DESCRIPTION
**Pull request by optimumtact**
Once added you cannot remove the chainsaw and the hand will be forever
used

This was coded by @PKPenguin321, I'm just comitting it to help them out

### CODE REVIEW

**Comment by daman997** [comment]
RIP AND TEAR YOUR GUTS!

File: code/modules/surgery/limb augmentation.dm [hunk 1]
```diff
@@ -74,3 +74,34 @@
...
+                user << "<span class='warning'>You can't fit the chainsaw in while [target]'s hands are full!</span>"
+                return 0
+        else
+                user.visible_message("[user] finshes installing the chainsaw!", "<span class='notice'>You install the chainsaw.</span>")
+                user.unEquip(tool)
+                qdel(tool)
+                var/obj/item/weapon/mounted_chainsaw/sawarms = new(target)
+                target.put_in_hands(sawarms)
+
+                    return 1
```
Comment (RemieRichards) [inline code/modules/surgery/limb augmentation.dm]: this return 1's indentation is fubar

**Comment by PKPenguin321** [comment]
i wrote this code yay

i just remembered it still needs unique sprites, right now it uses the normal chainsaw inhands

File: code/modules/surgery/limb augmentation.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/surgery/limb augmentation.dm]: fuq
it's normal in dreammaker, i guess i borked something when i moved it to the pastebin to send it to oranges

Comment (optimumtact) [inline code/modules/surgery/limb augmentation.dm]: herp my bad

**Comment by KorPhaeron** [comment]
@PKPenguin321 

Could probably just take the normal chainsaw inhand and rotate it 90 degrees

**Comment by PKPenguin321** [comment]
trust me, i tried
it's actually way too huge for that to work

**Comment by Incoming5643** [comment]
Fair warning about nodrop/abstract weapons: You can attack with them from the floor (provided you're not under a stun or anything.

**Comment by PKPenguin321** [comment]
that's the idea
see, the benefit of this is that once it's on your hand, you don't need two hands to do full damage with it and it can't be disarmed/can be used while lying down
the downside is that there is literally no way to get rid of it short of dying and getting cloned, meaning you basically only have one hand (not to mention people may attack/avoid you because you have an unconcealable chainsaw hand)

**Comment by Zilenan91** [comment]
but chainsaw hands so it's worth it

**Comment by KorPhaeron** [comment]
Lets not worry too much about the balance of chainsaw hands it sounds fun

**Comment by Shihrt** [comment]
Yup, same thing happened with augments. Everyone went to robotics to become THE LAW every round for a couple of weeks but then the hype died off.

**Comment by optimumtact** [comment]
Are there even that many chainsaws on the station?

**Comment by KorPhaeron** [comment]
@optimumtact

It's a single sheet of plasteel+a circ saw (autolathe)+cable coil.

Cargo can make like 50 of them roundstart with EVA supplies if they wanted

**Comment by optimumtact** [comment]
yolo

**Comment by Saegrimr** [comment]
Can you still cuff them? How does that work?

**Comment by RemieRichards** [comment]
@Saegrimr handcuffs aren't related to hands in the slightest (which is hilarious)

**Comment by PKPenguin321** [comment]
i was considering making it so that if you have at least one you can cut through handcuffs extra fast but i decided not to because a) i'm inexperienced with coding and don't want to touch anything i didn't make (like handcuff code) and b) i'd rather not make this too strong so it's not a balance concern

**Comment by Cheridan** [comment]
happening

**Comment by optimumtact** [comment]
https://www.youtube.com/watch?v=unnLg1TPCYM

**Comment by PKPenguin321** [comment]
yee

**Pull request merged**